environment.yml

```diff
@@ -0,0 +1,18 @@
+name: reconstructSPI
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - codecov
+  - coverage
+  - gemmi
+  - numba
```

Comment:
make sure you need all these dependencies. otherwise remove. I don't think numba is used right now, so take it out.